src/components/header/components/desktop/desktop.module.css

```diff
@@ -0,0 +1,84 @@
+.header {
+  display: flex;
+  padding: 8px 24px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.flex {
+  display: flex;
+}
+
+.list, .theme {
+  display: inline-flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.align_center {
+  align-items: center;
+}
+
+.color_white {
```

Comment:
for future need to read BEM methodology for class naming 
[https://en.bem.info/methodology/quick-start/](url)